src/1_Iterations/index.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Use the exercise name as the filename. It will make it easier to follow. Also, since this is Javascript the file name should be camelCased. i.e `src/iterations/binaryGap.js

Reply:
I added the number to the beginning of the folder name to denote that it is the first lesson